_docs_integrate/delete-attributes.md

```diff
@@ -111,8 +111,10 @@ Moreover, if the peer shared Attribute had a successor, its `succeeds` property
 Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) with a [PeerSharedAttributeDeletedByPeerNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem) is generated and sent to the owner of the peer shared Attribute, informing them that you deleted the Attribute they shared with you.
 Consequently, the `deletionInfo` of their corresponding own shared Attribute and of all potential predecessors is updated with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
 In case the owner already [deleted their own shared Attribute](#delete-own-shared-attributes), nothing happens.
-Please further note that the Notification is queued if the [Relationship is currently terminated]({% link _docs_integrate/terminate-relationships.md %}#terminate-an-active-relationship).
-It can then only be received and processed once the [Relationship has been reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).
+Please further note that the Notification is queued if the [Relationship is currently terminated]({% link _docs_integrate/terminate-relationships.md %}#terminate-an-active-relationship) but not yet decomposed.
+It can then only be received and processed if the [Relationship has been reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).
+The Notification is also queued if the [peer is currently in deletion]({% link _docs_integrate/delete-identities.md %}#effects-of-identity-deletion-on-relationships) but not yet deleted.
+It can then only be received and processed if the peer has [cancelled its deletion]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}).
```

Comment:
```suggestion
It can then only be received and processed if the peer [cancels its deletion]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}).
```